abcd/frontends/commandline/decorators.py

```diff
@@ -1,59 +1,57 @@
 import logging
-
+import functools
 from abcd import ABCD
-
 from abcd.frontends.commandline.config import Config
-from abcd.parsers.queries import parser
 
 logger = logging.getLogger(__name__)
 
 
 def init_config(func):
     config = Config.load()
 
+    @functools.wraps(func)
     def wrapper(*args, **kwargs):
         func(*args, config=config, **kwargs)
 
     return wrapper
 
 
 def init_db(func):
+    @functools.wraps(func)
     def wrapper(*args, config, **kwargs):
-        url = config.get('url', None)
+        url = config.get("url", None)
+        use_ssl = config.get("use_ssl", None)
 
         if url is None:
-            print('Please use abcd login first!')
+            print("Please use abcd login first!")
             exit(1)
```

Comment:
Do you want to `exit`? Seems risky to throw someone out of their python shell rather than raising.